packages/charts/src/renderers/line/line-accessors.ts

```diff
@@ -37,7 +37,7 @@ export class LineAccessors extends XYAccessors implements ILineAccessors {
     public data: ILineDataAccessors;
     public series: ILineSeriesAccessors;
 
-    constructor(private colorProvider = defaultColorProvider(), private markerProvider = defaultMarkerProvider()) {
+    constructor(public colorProvider = defaultColorProvider(), public markerProvider = defaultMarkerProvider()) {
```

Comment:
is changing them to public needed for some reason? doesn't hurt I guess.

Reply:
No reason just doing it for ease